.travis.yml

```diff
@@ -1,38 +1,48 @@
 dist: trusty
 sudo: false
 language: cpp
+os: linux
 
 matrix:
   include:
     # GCC 4.9
```

Comment:
Comments on compiler version are no longer necessary - the settings themselves are descriptive.